vividus-tests/src/main/resources/story/integration/NestedSteps.story

```diff
@@ -1,14 +1,9 @@
-Description: Integration tests for GenericNestedSteps class.
-
 Meta:
-    @epic vividus-extension-selenium
-
-
-Scenario: Verify step: When I find $comparisonRule `$number` elements `$locator` and while they exist do up to $iterationLimit iteration of$stepsToExecute
-Meta:
-    @requirementId 2054
-
-Given I am on a page with the URL '${vividus-test-site-url}/elementState.html'
-When I find > `0` elements `id(element-to-hide)` and while they exist do up to 10 iteration of
-|step                                             |
-|When I click on element located `id(button-hide)`|
+    @epic vividus-plugin-web-app
+
+Scenario: Verify step "I find $comparisonRule `$number` elements by `$locator` and for each element do$stepsToExecute"
+Given I am on a page with the URL 'https://demo.guru99.com/'
```

Comment:
we're changing our tests to do not depend on external resources, if our test site doesn't have needed functionality, please feel free to open a PR